cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -88,6 +89,20 @@ func (s *storageExt) Start(ctx context.Context, host component.Host) error {
 		s.factories[name] = factory
 	}
 
+	for name, g := range s.config.GRPC {
```

Comment:
Suggestion: could you extract changes to `jaegerstorage` into a new PR that we can merge separately? It's largely independent of what's blocking the integration tests.